content/blog/2022/07/vscode-camel-language-support-0.4.0-release/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Released version 0.4.0 of VSCode Language Support for Apache Camel"
+date: 2022-07-21
+authors: [joshiraez]
+categories: ["Tooling", "Releases"]
+preview: "Released version 0.4.0 of VSCode Language Support for Apache Camel. New completion to add Camel K modeline, updates on the default catalogs and support for the productized version."
+---
+
+# Release 0.4.0
+
+This new release adds a new completion to add Camel K modeline to new files.
+It also brings updates on the default Camel catalogs and brings support for the productized classic Camel catalog.
+
+# Insert Camel K Modeline completion
+
+This completion allows you to add Camel K Modeline at the beginning of your Camel files.
+
+![Insert Camel K Modeline completion](./img/ModelineInsertionBlog.gif)
+
+# Catalog upgrades
+
+This new release contains the following upgrades:
+
+* Camel Quarkus Catalog from 2.7.0 to 2.8.0
+* Default Camel Catalog version from 3.15.0 to 3.16.0
+* Kamelet Catalog from 0.7.1 to 0.8.0
+* Support for the productized Camel catalog (not Kamelets)
```

Comment:
thi is not a catalog upgrade. it is a different and new feature which is worth its own chapter I guess